frontend/src/features/admin-form/create/constants.ts

```diff
@@ -361,3 +361,24 @@ export const MYINFO_FIELD_TO_DRAWER_META: {
     isSubmitted: true,
   },
 }
+// to remove after 1 July. For testing purposes, deliberately setting target date as 1 day before today (10 Jun) to simulate change
```

Comment:
Can we mark it as a `// TODO:`

We have a convention of using `// TODO:` to mark things to complete at a later time. (actually, we have too many todos 😆 )